pkg/spdx/package.go

```diff
@@ -202,13 +236,28 @@ func (p *Package) Render() (docFragment string, err error) {
 				return docFragment, errors.New("unable to render package, files were analyzed but some do not have sha1 checksum")
 			}
 			shaList = append(shaList, f.Checksum["SHA1"])
+
+			// Collect the license tags
+			if f.LicenseInfoInFile != "" {
+				filesTagList[f.LicenseInfoInFile] = nil
+			}
 		}
 		sort.Strings(shaList)
 		h := sha1.New()
 		if _, err := h.Write([]byte(strings.Join(shaList, ""))); err != nil {
 			return docFragment, errors.Wrap(err, "getting sha1 verification of files")
 		}
 		p.VerificationCode = fmt.Sprintf("%x", h.Sum(nil))
+
+		for tag := range filesTagList {
+			if tag != "NONE" && tag != "NOASSERTION" {
```

Comment:
Let's make `NONE` a const somewhere.

Reply:
Done, pushed another commit moving `NONE` and `NOASSERTION` to constants as probably they will show up a lot